pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -1377,4 +1377,10 @@ self: super: builtins.intersectAttrs super {
   pvar = dontCheck super.pvar;
 
   kmonad = enableSeparateBinOutput super.kmonad;
+
+  # cabal lib set as unbuildable in linux so callCabal2nix generates a dummy derivation
+  jsaddle-wkwebview = overrideCabal (drv: {
+    libraryFrameworkDepends = with pkgs.buildPackages.darwin.apple_sdk.frameworks; [ Cocoa WebKit ];
+    libraryHaskellDepends = with self; [ aeson data-default jsaddle ];
+  }) super.jsaddle-wkwebview;
```

Comment:
Hm, this is a corner case I am uncertain about. But I think since this is a platform specific override I’d prefer it to go to `configuration-darwin.nix`. Sorry, for first pointing you to this file.